### DISCUSSION TITLE
Trailing slash for APIs

### DISCUSSION THREAD

**Discussion by lbarad** [rahulvemula/SE-leasingPortal#99] [Q&A]
I think API routes should work without the ending '/'

**Comment by vamsibvv** [comment]
Ok modified it. please validate

**Comment by vamsibvv** [comment]
Changed all rest apis to /users - for get all users and /users/{email} for get user by email (basically added 's at the end of rest apis)

**Comment by lbarad** [comment]
@vamsibvv  Do you mind changing the /users API to /user-list, given it returns the entire list of users?

**Comment by vamsibvv** [comment]
I think /users lgtm as /user-list is confusing a bit.
